src/components/MaterialUI/Inputs/DatePicker.js

```diff
@@ -131,12 +134,18 @@ const WrappedDatePicker = ({
 	...props
 }) => {
 	const classes = useStyles({ readOnly });
-	const startDate = value ? new Date(value) : null;
+	const startDate = value ? setLocalZoneDate(new Date(value), getTimeZoneByName(timePickerTimeZone)) : null;
```

Comment:
define and calculate timeZoneName once and then use
const timeZoneName = getTimeZoneByName(timePickerTimeZone);